docs/sources/static/configuration/create-config-file.md

```diff
@@ -42,7 +42,7 @@ integrations:
 ```
 
 In this example, we first must configure the `wal_directory` which is used to
-store metrics in a Write-Ahead Log. This is required, but ensures that samples
+store metrics in a Write-Ahead Log (WAL). WAL is required and ensures that samples
```

Comment:
nit: Should this say "The WAL is required?"

Reply:
Fixed in the next commit.